tests/unit_tests/cpu/tools_navigator.cpp

```diff
@@ -289,7 +289,7 @@ GTEST_TEST(detray_navigator, wire_chamber) {
 
     // test track
     point3 pos{0.f, 0.f, 0.f};
-    vector3 mom{1.f, 0.f, 0.f};
+    vector3 mom{0.f, 1.f, 0.f};
```

Comment:
I changed to test (0,1,0) direction rather than (1,0,0).
As the wire chamber covers [0, 2pi] there is always a crosssection overlap between two cells at zero phi. This makes two wire surface intersections with (0,1,0) direction, not a catastrophe but I would not like to test this special case